waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -29,12 +29,16 @@ $any = [.$white]
 @double = "-"? $digit+ "." $digit+
 @integer = "-"? $digit+
 @ident = $identstart $ident* "'"*
+@linecomment = "//" [^\n\r]*
+@blockcomment = "/*" (("*"[^\/]) | [^\*] | $white)* "*/" -- Based on https://stackoverflow.com/a/16165598/1509394 .
 
 -- Tokenization rules (regex -> token)
 tokens :-
 
--- Skips whitespace
+-- Skips whitespace and comments
 <0>       $white+ ;
+<0>       @linecomment ;
+<0>       @blockcomment ;
```

Comment:
This is all that was needed to add line `//` and block `/* */` comments to Wasp! I forgot to add them before :D so I added them here.

We just discard them -> if we wanted to store them into AST that would be quite more work.